sql.txt

```diff
@@ -1,14 +1,49 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
+------------------------------------------------------------------
+Assuming 'Entry 1' is Row 0 (which is the first Row):
+SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country LIMIT 10 OFFSET 10;
+
+However if assuming 'Entry 1' is Row 1 (which is the 2nd Row):
+SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country LIMIT 10 OFFSET 11;
+------------------------------------------------------------------
+
 
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT * FROM Customers WHERE ContactName LIKE 'A%';
+
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT * FROM OrderDetails WHERE ProductID = 51 AND quantity > 10;
+
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
+----------------------------------------------------------------
+With ProductID auto-incrementing, the three SQL statements are
+INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price) VALUES ("Dan-O's Seasoning (Original, Spicy & Hot Chipotle)", 1, 2, "3 pack of 3.5 oz bottles", 28.44);
+
+INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price) VALUES ("Aplenty Organic Garam Masala Seasoning Blend", 1, 2, "1.8 oz bottle", 3.99);
+
+INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price) VALUES ("Slap Ya Mama All Natural Cajun Seasoning from Louisiana", 1, 2, "8 oz bottle", 2.99);
+
+N.B. The above three SQL statements can be combined into a single SQL statement as follows:
+INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price) VALUES 
+("Dan-O's Seasoning (Original, Spicy & Hot Chipotle)", 1, 2, "3 pack of 3.5 oz bottles", 28.44),
+("Aplenty Organic Garam Masala Seasoning Blend", 1, 2, "1.8 oz bottle", 3.99),
+("Slap Ya Mama All Natural Cajun Seasoning from Louisiana", 1, 2, "8 oz bottle", 2.99);
+----------------------------------------------------------------
+
 
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
 
+UPDATE Products SET Price = Price + 1.50 WHERE ProductID IN (SELECT ProductID FROM Products LIMIT 2);
```

Comment:
Something that is good to know for the future: Not all databases will return results in the order that they appear in the database. If the order is important, using `ORDER BY` will ensure that the results are in a specific order.